docs/symptom-survey/contingency-tables.md

```diff
@@ -90,34 +90,131 @@ alphabetically. Each aggregate reports four columns (unrounded):
+| `pct_hesitant_barrier_health_condition` | Estimated percentage of respondents who say they are hesitant to get vaccinated because they have a health condition that may impact the safety of a COVID-19 vaccine, among respondents who answered No, probably not" or "No, definitely not" when asked if they would get vaccinated if offered (item V3). This series of items was shown to respondents starting in Wave 8. <br/> **Earliest date available:** 2021-02-08 | V5bc |
+| `pct_hesitant_barrier_pregnant` | Estimated percentage of respondents who say they are hesitant to get vaccinated because they are pregnant or breastfeeding, among respondents who answered No, probably not" or "No, definitely not" when asked if they would get vaccinated if offered (item V3). This series of items was shown to respondents starting in Wave 8. <br/> **Earliest date available:** 2021-02-08 | V5bc |
+| `pct_hesitant_barrier_religious` | Estimated percentage of respondents who say they are hesitant to get vaccinated because it is against their religious beliefs, among respondents who answered No, probably not" or "No, definitely not" when asked if they would get vaccinated if offered (item V3). This series of items was shown to respondents starting in Wave 8. <br/> **Earliest date available:** 2021-02-08 | V5bc |
+| `pct_hesitant_barrier_other` | Estimated percentage of respondents who say they are hesitant to get vaccinated for another reason, among respondents who answered No, probably not" or "No, definitely not" when asked if they would get vaccinated if offered (item V3). This series of items was shown to respondents starting in Wave 8. <br/> **Earliest date available:** 2021-02-08 | V5bc |
+
+The set of "barrier" items correspond to the set of "hesitancy_reasons" items in the API.
+
+### Reasons for Believing Vaccine is Unnecessary
+
+Respondents who indicate that "I don't believe I need a COVID-19 vaccine" (in
+items V5a, V5b, V5c, or V5d) are asked a follow-up item asking why they don't
+believe they need the vaccine. These signals summarize the reasons selected.
+Respondents who do not select any reason (including "Other") are treated as
+missing.
+
+This item was shown to respondents starting in Wave 8.
+
+| Indicator | Description | Survey Item |
+| --- | --- | --- |
+| `pct_hesitant_dontneed_reason_had_covid` | Estimated percentage of respondents who say they don't need to get a COVID-19 vaccine because they already had the illness, among respondents who answered No, probably not" or "No, definitely not" when asked if they would get vaccinated if offered (item V3) and provided at least one reason for why they believe a COVID-19 vaccine is unnecessary. <br/> **Earliest date available:** 2021-02-08 | V6 |
```

Comment:
Two things:

1. Missing quotation mark on "No, probably not" for all of these
2. Are these really only among "No" respondents, and not "Yes, probably"? Because V6 is shown to people who answer "Yes, probably" but say they don't need the vaccine.

Reply:
Yes, this is only among "No"/hesitant respondents. To calculate this indicator, I nulled out responses from people who'd gotten to V6 from V5a or V5d